test/e2e/test_tools.py

```diff
@@ -136,57 +214,30 @@ def test_get(self) -> None:
+        lists = project_expected["project"]["lists"]
+        test_list: dict[str, str] = next((l for l in lists if l["name"] == "testlist"), {})
+        not_used_list: dict[str, str] = next((l for l in lists if l["name"] == "notUsedList"), {})
+        excel_list: dict[str, str] = next((l for l in lists if l["name"] == "my-list-from-excel"), {})
 
-        lists_out = test_project_out['project']['lists']
-        test_list_out: dict[str, str] = next((l for l in lists_out if l['name'] == 'testlist'), {})
-        not_used_list_out: dict[str, str] = next((l for l in lists_out if l['name'] == 'notUsedList'), {})
-        excel_list_out: dict[str, str] = next((l for l in lists_out if l['name'] == 'my-list-from-excel'), {})
+        lists_out = project_received["project"]["lists"]
+        test_list_out: dict[str, str] = next((l for l in lists_out if l["name"] == "testlist"), {})
+        not_used_list_out: dict[str, str] = next((l for l in lists_out if l["name"] == "notUsedList"), {})
+        excel_list_out: dict[str, str] = next((l for l in lists_out if l["name"] == "my-list-from-excel"), {})
 
-        self.assertEqual(test_list.get('labels'), test_list_out.get('labels'))
-        self.assertEqual(test_list.get('comments'), test_list_out.get('comments'))
-        self.assertEqual(test_list.get('nodes'), test_list_out.get('nodes'))
+        self.assertEqual(test_list.get("labels"), test_list_out.get("labels"))
+        self.assertEqual(test_list.get("comments"), test_list_out.get("comments"))
+        self.assertEqual(test_list.get("nodes"), test_list_out.get("nodes"))
 
-        self.assertEqual(not_used_list.get('labels'), not_used_list_out.get('labels'))
-        self.assertEqual(not_used_list.get('comments'), not_used_list_out.get('comments'))
-        self.assertEqual(not_used_list.get('nodes'), not_used_list_out.get('nodes'))
+        self.assertEqual(not_used_list.get("labels"), not_used_list_out.get("labels"))
+        self.assertEqual(not_used_list.get("comments"), not_used_list_out.get("comments"))
+        self.assertEqual(not_used_list.get("nodes"), not_used_list_out.get("nodes"))
 
-        self.assertEqual(excel_list.get('comments'), excel_list_out.get('comments'))
+        self.assertEqual(excel_list.get("comments"), excel_list_out.get("comments"))
 
-    def test_excel_to_json_list(self) -> None:
-        excel_to_json_lists.excel2lists(excelfolder='testdata/lists_multilingual',
-                                        path_to_output_file='testdata/tmp/_lists-out.json')
 
-    def test_excel_to_json_resources(self) -> None:
-        excel2resources(excelfile='testdata/Resources.xlsx',
-                        path_to_output_file='testdata/tmp/_out_resources.json')
-
-    def test_excel_to_json_properties(self) -> None:
-        excel2properties(excelfile='testdata/Properties.xlsx',
-                         path_to_output_file='testdata/tmp/_out_properties.json')
```

Comment:
I put these three to the end, so that the file is ordered according the  order in `dsp_tools.py`